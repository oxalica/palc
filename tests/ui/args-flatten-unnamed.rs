use palc::Args;

#[derive(Args)]
struct Outer {
    #[command(flatten)]
    inner: Inner,
}

#[derive(Args)]
struct Inner {
    unnamed: String,
}

fn main() {}
