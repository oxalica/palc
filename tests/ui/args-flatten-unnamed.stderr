error[E0080]: evaluation panicked: TODO: cannot arg(flatten) positional arguments yet
 --> tests/ui/args-flatten-unnamed.rs:6:12
  |
6 |     inner: Inner,
  |            ^^^^^ evaluation of `_` failed here
